docs/pages/versions/unversioned/sdk/reanimated.md

```diff
@@ -22,23 +22,11 @@ The second major version of this library offers a much easier API, along with si
 
 > ⏩ If you want to play with v2-alpha in a new project before adding it to an existing project, run `npx crna --template with-reanimated2` to create a project with it configured and ready to use.
 
-To opt in to using TurboModules for react-native-reanimated, add the `experiments.turboModules` key to your `app.json`:
```

Comment:
is it no longer needed to enable turboModules explicitly with this flag?

Reply:
That is correct, the reanimated team has changed the way their native code is integrated and as they exposed a property on `global` on Android in SDK39, they do so now on iOS too.

See first note at https://docs.swmansion.com/react-native-reanimated/docs/installation#ios

Reply:
amazing! thanks for the info